pkg/errdefs/errors.go

```diff
@@ -40,3 +40,12 @@ func isErrConnectionRefused(err error) bool {
 func NeedsRetryWithHTTP(err error) bool {
 	return err != nil && (isErrHTTPResponseToHTTPSClient(err) || isErrConnectionRefused(err))
 }
+
+func isErrDifferentFsVersion(err error) bool {
+	const errMessage = "exit status 2"
```

Comment:
We should check the exit code instead of error message, example https://stackoverflow.com/questions/10385551/get-exit-code-go